ONSdigital/eq-author-app
# EAR 1380 - Dependabots (Sprint 54)

### What is the context of this PR?

Dependabot upgrades for Sprint 54.

[Ticket](https://collaborate2.ons.gov.uk/jira/browse/EAR-1380)

### How to review

- `yarn`, `yarn`, `yarn`
- Run tests
- Check builds pass

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
